wd_helper.au3

```diff
@@ -1343,6 +1343,9 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 			If $iErr = $_WD_ERROR_Success Then
 				Local $bDriverExists = ($sDriverCurrent <> 'None')
 
+				; Determine if the remote and local files are the same size, just in case they are different, force a download. This is needed because 32/64 bit versions have different file sizes.
+				If FileGetSize($sInstallDir & $sDriverEXE) <> InetGetSize($sURLNewDriver, $INET_FORCERELOAD) Then $bForce = True
```

Comment:
Wouldn't it make sense to change this to `If $bDriverExists And FileGetSize($sInstallDir & $sDriverEXE) <> InetGetSize($sURLNewDriver, $INET_FORCERELOAD) Then $bForce = True`? No point in checking the file sizes if we know that the local file doesn't exist.

Reply:
Done

Reply:
In hindsight, this won't work because you are comparing the size of the local EXE to the remote ZIP, which will never match.

Reply:
reverted